admin/src/components/CreationFormular.vue

```diff
@@ -298,6 +283,36 @@ export default {
         })
     },
   },
+  computed: {
+    currentMonth() {
+      return {
+        short: this.$d(this.now, 'month'),
+        long: this.$d(this.now, 'short'),
+        year: this.$d(this.now, 'year'),
+        date: this.$d(this.now, 'short', 'en'),
+      }
+    },
+    lastMonth() {
+      const now = new Date(this.now)
+      const lastMonth = new Date(now.getFullYear(), now.getMonth() - 1, 1)
```

Comment:
What happens if now is 01.01.2022 does last Month change to 01.12.2021?

Reply:
The test written below seems to say yes

Reply:
Yes. You can try that out on the console.

Reply:
```
const date = new Date('2022-01-01')
Date Sat Jan 01 2022 01:00:00 GMT+0100 (Central European Standard Time)
const newDate = new Date(date.getFullYear(), date.getMonth() -1, 1)
Date Wed Dec 01 2021 00:00:00 GMT+0100 (Central European Standard Time)
```